src/lib/directive.js

```diff
@@ -68,7 +68,7 @@ const _ = require('lodash');
         return target.email;
       }).join(',');
 
-      return new URL('/mailto?uri=mailto:' + encodeURIComponent(recipients), $window.location.origin).toString();
+      return new URL('/mailto/?uri=mailto:' + encodeURIComponent(recipients), $window.location.origin).toString();
```

Comment:
do we need another `/mailto` like this `return new URL('/mailto/mailto/?uri=mailto:' + encodeURIComponent(recipients), $window.location.origin).toString();`